libs/sdk-ui-dashboard/src/presentation/topBar/editModeButton/useEditModeButtonHandlers.ts

```diff
@@ -0,0 +1,14 @@
+// (C) 2022 GoodData Corporation
+
+import { IEditModeButtonHandlers } from "./types";
+import { uiActions, useDashboardDispatch } from "../../../model";
+import { useCallback } from "react";
+
+export function useEditModeButtonHandlers(): IEditModeButtonHandlers {
```

Comment:
Should this rather be `useDashboardRenderMode` so we can use it later for the cancel button and make it usable also for the plugins?